src/utils/tensorrt_inference.py

```diff
@@ -0,0 +1,178 @@
+import glob
```

Comment:
maybe rename as `predict_model_tensorrt.py` or something to standardize it with the other scripts?

Reply:
will rename it.